go.mod

```diff
@@ -47,6 +46,11 @@ require (
 	gorm.io/plugin/dbresolver v1.4.1
 )
 
+require (
+	github.com/golang-jwt/jwt/v4 v4.5.0 // indirect
+	github.com/golang-jwt/jwt/v5 v5.0.0
+)
+
```

Comment:
S: merge this into the above.

Reply:
> S: merge this into the above.

`github.com/Azure/go-autorest/tree/main/autorest` require v4 version. and `github.com/Azure/go-autorest/tree/main/autorest` require by `github.com/golang-migrate/migrate/v4/database/sqlserver`